### DISCUSSION TITLE
Routing interception

### DISCUSSION THREAD

**Discussion by EnJay240** [nuxt/nuxt#10482] [Questions (Nuxt 2)]
Hello dear community 😇
For some time now I've been puzzling over how to implement this function:
I need to intercept a transition by a link, for example, I have:
`<div v-html="variable" />`
Multiple tags are stored in the middle `<a href="url">url</a>`
But at the same time, I need to make it so that when clicking on each individual `<a>` call the Vue method, without following the link. That is to intercept the transition to the site. Is it possible to do this?
I will be grateful for advice ☺️

**Comment by vinayakcharodi** [comment]
This feature not available in Nuxt for now.

**Comment by sabeerbikba** [comment]
I have just implemented a fully working production-ready example of **Intercepting Routes** (similar to Next.js `(.)` syntax) for Nuxt 3.

It handles the edge cases that usually break this pattern:
✅ **Deep Linking:** Refreshing the page loads the standalone page (SSR friendly).
✅ **Soft Navigation:** Clicking an item opens the modal without reloading.
✅ **History API:** The Browser Back button closes the modal correctly.
✅ **SEO Optimized:** Uses real `<a>` tags so crawlers can index the content.

It uses a Composable + Conditional Rendering strategy rather than a complex router hack.

**Demo & Code:**
https://github.com/sabeerbikba/nuxt-intercepting-routes

Hope this helps anyone struggling with the "Instagram-style" routing pattern!

**Reply by cernymatej** [reply]
I think your demo is private

**Reply by sabeerbikba** [reply]
I corrected it
